### DISCUSSION TITLE
Re-ordering snippets

### DISCUSSION THREAD

**Discussion by Lodan-Max** [isontheline/pro.webssh.net#653] [Ideas]
I frequently make use of the snippets in WebSSH. To update a pi, or to reboot a device.
It looks like WebSSH randomizes the placement of snippets. working on autopilot I more than once rebooted a machine instead of requesting an update.

A simple idea would be to rearrange them yourself so they would appear inside the shell and in the snippet-window in the order you could use them.

**Comment by isontheline** [comment]
Hello @Lodan-Max 👋

Thanks for your feedback 🙏

I'm fixing this issue inside the upcoming release : #649

**Reply by isontheline** [reply]
@Lodan-Max The fix will sort in alphabetical order. 

I need more time to implement the self rearrange because snippets are linked by tags and not all snippets are showed every time.

**Reply by isontheline** [reply]
@Lodan-Max #649 Now fixed in 19.1.919 : 
https://testflight.apple.com/join/QSrBK59z

**Reply by Lodan-Max** [reply]
Alphabetical order seems to work. Can be arranged by numbering them for now.
